docs/dev/development.md

```diff
@@ -381,6 +381,9 @@ it creates rpm packages after a DAOS build. Regardless of how that build is done
 it will put files in the right places in the final packages.  Most of the magic
 is in [`utils/rpms/fpm_common.sh`](../../utils/rpms/fpm_common.sh) with component specific code in
 `utils/rpms/<component>.sh`.
+A dedicated changelog should be maintained for each component in
+the `utils/rpms/<component>.changelog` file and should be referenced by
+the `RPM_CHANGELOG="<component>.changelog"` variable in `utils/rpms/<component>.sh`.
```

Comment:
```suggestion
```

I think this paragraph is complicated enough. IMHO we can get away with a dedicated point below.